Config.uk

```diff
@@ -170,6 +170,10 @@ config LWIP_UDP
 	bool "UDP support"
 	default y
 
+menuconfig LWIP_RAW
+	bool "RAW support"
+	default n
+
```

Comment:
I think we should add a `help` for this option. lwIP also provides a RAW/low-level API to program against but I think this option is different according to the docs: `Enable application layer to hook into the IP layer itself.` Probably it enables `SOCK_RAW` but are there other APIs available then too or should this option depend on sockets?

Reply:
I've updated the help text with your recommendation.